src/Lykke.Service.BitfinexAdapter.Core/Domain/Exchange/ExchangeState.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Lykke.Service.BitfinexAdapter.Core.Domain.Exchange
+{
+    public enum ExchangeState
```

Comment:
These Exchange states are redundant I think. Basically, the Exchange (ExchangeBase, BitfinexExchange) class is only used for api calls. It doens handle/manage any web socket connections and its Start method doenst do anything meaningful, apart from setting the state to Connected which is later not used anywhere (there arent multiple instances of the exchange class.) WebSocket connection is used in OrderBookHarvester for receiving orderbooks - there only one instance of it using a single WebSocket  connection (there is no point having separate connection per client here). TradeExecution however (BitfinexExecutionHarvester instance per client) has a separate connection per client, because trades are client specific. Anyway, none of these classes uses ExchangeState, so I will remove it (or probably just leave the Connected state).